src/operator/quantization/quantized_fully_connected.cc

```diff
@@ -79,6 +80,20 @@ bool QuantizedFullyConnectedType(const nnvm::NodeAttrs& attrs,
   return true;
 }
 
+bool QuantizedFullyConnectedStorageType(const nnvm::NodeAttrs& attrs,
+                                        const int dev_mask,
+                                        DispatchMode* dispatch_mode,
+                                        std::vector<int> *in_attrs,
+                                        std::vector<int> *out_attrs) {
+  *dispatch_mode = DispatchMode::kFCompute;
+  if (dev_mask == mshadow::cpu::kDevMask) {
+    *dispatch_mode = DispatchMode::kFComputeEx;
+  }
+  for (size_t i = 0; i < out_attrs->size(); i++)
+    (*out_attrs)[i] = kDefaultStorage;
+  return true;
```

Comment:
What if `in_attrs` has unknown storage types? You need to
1. Check and assign stype to `in_attrs` as well.
2. return false if any stype is unknown in `in_attrs` and `out_attrs`.

Reply:
fixed

Reply:
Please consider using range for loops for readability.

Reply:
fixed

Reply:
I think @larroy meant to use:
```
for (auto &v : in_attrs) {
  // ...
}
```

Reply:
fixed